pyband/messages/band/tss/v1/msgs.py

```diff
@@ -0,0 +1,91 @@
+from pyband.messages.base import BaseMessageWrapper
+from pyband.proto.band.tss.v1beta1 import (
+    MsgSubmitDkgRound1 as MsgSubmitDkgRound1Proto,
+    MsgSubmitDkgRound2 as MsgSubmitDkgRound2Proto,
+    MsgComplain as MsgComplainProto,
+    MsgConfirm as MsgConfirmProto,
+    MsgSubmitDEs as MsgSubmitDEsProto,
+    MsgResetDe as MsgResetDeProto,
```

Comment:
it should be `DE`.

Reply:
seems it's auto-generated. I'm not sure why MsgSubmitDEs got the correct one.

Reply:
its auto-generated, I can manually change it for now